spec/services/disclosure_log_spec.rb

```diff
@@ -93,7 +93,7 @@
     let!(:published_request4) do
       create(:published_request,
              payload: { ref: 'FOI-4',
-                        datepublished: Time.zone.parse('2018-06-17'),
+                        datepublished: '',
```

Comment:
Comment says "It has an empty datepublished so will be deleted" – but `datepublished` is not blank. The spec passes, so that doesn't seem right. Changing it to what the comment says it should be also passes, so something doesn't seem quite right here.

Reply:
Oh, no, I'm talking nonsense.

Here we're creating the record that will pre-exist in the database. The empty `datepublished` refers to the attributes that come back from the API in `let(:requests)`.